internal/stack/options.go

```diff
@@ -11,6 +11,9 @@ type Options struct {
 	DaemonMode   bool
 	StackVersion string
 
+	// ref overrides
+	KibanaRefOverride string
```

Comment:
I suggest not exposing additional flag for Kibana as we will for sure need more like this in the feature.

Actually there might be a simpler way to do this, via `stack.ImageRefOverridesForVersion`. You can try to modify **only** this function to read special ENV vars: `ELASTIC_AGENT_IMAGE_REF_OVERRIDE`, `KIBANA_IMAGE_REF_OVERRIDE`, etc. and give it priority to `image_ref_overrides` from `config.yml`.

BTW this `config.yml` is yet another way to globally overide image references. It's used to replace faulty images in case of emergency)

Reply:
I removed the flag and used an env var to get the custom kibana ref. Please let me know your thoughts, thanks!

Reply:
@mtojek wait, I did not read it properly. Will work on a new commit to modify ONLY `ImageRefOverridesForVersion `. Sorry!

Reply:
I think it's now ready, thanks for the review!